modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/internal/model/listener/test/UserModelListenerTest.java

```diff
@@ -107,14 +110,27 @@ public void tearDown() throws Exception {
 	public void testOnAfterRemove() throws Exception {
 		User user = UserTestUtil.addUser();
 
-		ObjectDefinition objectDefinition =
+		ObjectDefinition objectDefinition1 =
 			_objectDefinitionLocalService.addCustomObjectDefinition(
 				user.getUserId(), 0, null, false, false, false, false,
 				LocalizedMapUtil.getLocalizedMap("Able"), "Able", null, null,
 				LocalizedMapUtil.getLocalizedMap("Ables"), true,
 				ObjectDefinitionConstants.SCOPE_COMPANY,
 				ObjectDefinitionConstants.STORAGE_TYPE_SALESFORCE,
 				Collections.emptyList());
+		ObjectDefinition objectDefinition2 =
+			ObjectDefinitionTestUtil.addCustomObjectDefinition(
+				ObjectDefinitionTestUtil.getRandomName(), user.getUserId());
+
+		ObjectRelationship objectRelationship =
+			_objectRelationshipLocalService.addObjectRelationship(
+				null, user.getUserId(),
+				objectDefinition1.getObjectDefinitionId(),
+				objectDefinition2.getObjectDefinitionId(), 0,
+				ObjectRelationshipConstants.DELETION_TYPE_PREVENT, false,
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				"a" + RandomTestUtil.randomString(), false,
+				ObjectRelationshipConstants.TYPE_ONE_TO_MANY, null);
```

Comment:
@marcelabc you could use the `ObjectRelationshipTestUtil` [here](https://github.com/liferay/liferay-portal/blob/0ac31e5e7dbc796ec7def4b357c17084ee4d485c/modules/apps/object/object-test-util/src/main/java/com/liferay/object/test/util/ObjectRelationshipTestUtil.java#L34-L37)